app/api/orders.py

```diff
@@ -438,8 +434,7 @@ def after_update_object(self, order, data, view_kwargs):
                                                      order.event.name, order.identifier)
 
     # This is to ensure that the permissions manager runs and hence changes the kwarg from order identifier to id.
-    decorators = (jwt_required, api.has_permission(
-        'auth_required', methods="PATCH,DELETE", model=Order),)
```

Comment:
You've now removed the option of patching orders

Reply:
@iamareebjamal Nopes. Tested.

Reply:
Tested what? Can a user PATCH an order created by itself and only itself?

Reply:
@iamareebjamal Yes.

https://github.com/fossasia/open-event-server/blob/931127b7edb2c22e7129043e081ef918df6ef4b6/app/api/orders.py#L324